packages/frontend/src/components/message/Message.tsx

```diff
@@ -1114,13 +1114,40 @@ function WebxdcMessageContent({
   tabindexForInteractiveContents: -1 | 0
 }) {
   const tx = useTranslationFunction()
+  const [webxdcInfo, setWebxdcInfo] = useState<T.WebxdcMessageInfo | null>(null)
+  const [isLoadingWebxdcInfo, setIsLoadingWebxdcInfo] = useState(false)
+  const accountId = selectedAccountId()
+
+  useEffect(() => {
```

Comment:
Looks like we need to listen on `MsgsChanged` around here, because otherwise the message info doesn't update live, e.g. with the Editor app. Try sending one, and then typing something. After that, the updates will only appear when you switch the chat.

And maybe the same goes for all other places where apps are displayed, e.g. gallery?

![image](https://github.com/user-attachments/assets/7748e32c-c2f8-449e-a086-5e565b74ada0)

Reply:
I added a listener for event WebxdcStatusUpdate 554c6bea556830990a3e23e12f6a244af312d562

Reply:
The same applies to mediaAttachement in Gallery where the content is also not updated when WebxdcStatusUpdate events happen, but that could maybe adresses in a followup since the use case to have the app and gallery open at the same time is probably more rare